src/Orbit.js

```diff
@@ -22,7 +30,8 @@ export const OrbitType = Object.freeze({
   PARABOLIC: 1,
   HYPERBOLIC: 2,
   ELLIPTICAL: 3,
-  UNKNOWN: 4,
+  TABLE: 4,
+  UNKNOWN: 5,
```

Comment:
Perhaps we should set UNKNOWN to 0 now.  If we ever were to serialize OrbitType in the future for some reason, it would be nice if the enum remained stable.

Reply:
Yep I can see that.